lib/problems.js

```diff
@@ -0,0 +1,26 @@
+
+
+const balanced = (string) => {
+    stack = []
+    const bracketMap = {
+        '}' : '{',
+        ']' : '[',
+        ')' : '('
+    }
+    index = 0
+
+    for (let index = 0; index < string.length; index++) {
+    if (string[index] in bracketMap) {
+        stack.push(string[index]); 
+        }
+        // compare bracket to hash value
```

Comment:
for this one you'd need to check to see if the character is a key in the object.  If so, pop and compare to make sure they match, if not then you can push the current character onto the stack.